src/scripts/GameConstants.ts

```diff
@@ -74,5 +90,59 @@ namespace GameConstants {
         "mediumfast",
         "fast"
     }
+
+    // Map navigation
+    export const AMOUNT_OF_ROUTES = 25;
+
+    /**
+     * Each route contains a list of routenumbers that need to be completed
+     * Access with routeRequirements.region.routeNumber
+     */
+    export const routeRequirements = {
+        0: {
+            1: [],
+            2: [1],
+            3: [2],
+            4: [3],
+            5: [4],
+            6: [5],
+            7: [5, 10],
+            8: [5, 6, 7],
+            9: [4],
+            10: [9],
+            11: [6, 12],
+            12: [7, 10, 11],
+            13: [11, 12],
+            14: [13],
+            15: [14],
+            16: [8],
+            17: [16],
+            18: [17],
+            19: [15, 18],
+            20: [19],
+            21: [20],
+            22: [1],
+            23: [22],
+            24: [4],
+            25: [24],
+        }
+    };
+
+    export const routeBadgeRequirements = {
+        0: {
+            3: GameConstants.Badge.Boulder,
```

Comment:
the accesToRoute method does not seem to work when the index is not specified (so for example route 1,2,4 etc)

`Uncaught TypeError: Cannot read property '1' of undefined
    at Function.RouteRequirement.accessToRoute (script.min.js:2900)
    at <anonymous>:1:18`